jetty-util/src/main/java/org/eclipse/jetty/util/resource/URLResource.java

```diff
@@ -48,7 +48,7 @@
     protected URLResource(URL url, URLConnection connection)
     {
         _url = url;
-        _urlString=_url.toExternalForm();
+        _urlString=_url.toExternalForm().replaceAll("/+|\\\\+", "/");
```

Comment:
It replaces any multiple (back|forward)slash with a single slash. 
If I remember correctly, a similar clean up was done in other parts of the class implementation.